e2e/e2e_test.go

```diff
@@ -150,64 +155,78 @@ func Test_HBAR(t *testing.T) {
 		t.Fatalf("[%s] - Failed to encode the authorisation signature. Error: [%s]", expectedTxRecord.TransactionID, err)
 	}
 
-	time.Sleep(20 * time.Second)
```

Comment:
I think we will need a more stable fix, as we already sync we could have a re-try in case of timeout.